src/main/resources/assets/securitycraft/lang/en_us.json

```diff
@@ -370,53 +370,53 @@
 	"help.securitycraft.keycard_lv1.info": "The keycards are used with the keycard reader. There are six keycards, level 1, 2, 3, 4, and 5 cards, and a limited use card that is equivalent to a level 5 card that can only be used five times in Survival mode.",
 	"help.securitycraft.keycard_lv2.info": "The keycards are used with the keycard reader. There are six keycards, level 1, 2, 3, 4, and 5 cards, and a limited use card that is equivalent to a level 5 card that can only be used five times in Survival mode.",
 	"help.securitycraft.keycard_lv3.info": "The keycards are used with the keycard reader. There are six keycards, level 1, 2, 3, 4, and 5 cards, and a limited use card that is equivalent to a level 5 card that can only be used five times in Survival mode.",
 	"help.securitycraft.keycard_lv4.info": "The keycards are used with the keycard reader. There are six keycards, level 1, 2, 3, 4, and 5 cards, and a limited use card that is equivalent to a level 5 card that can only be used five times in Survival mode.",
 	"help.securitycraft.keycard_lv5.info": "The keycards are used with the keycard reader. There are six keycards, level 1, 2, 3, 4, and 5 cards, and a limited use card that is equivalent to a level 5 card that can only be used five times in Survival mode.",
 	"help.securitycraft.limited_use_keycard.info": "The keycards are used with the keycard reader. There are six keycards, level 1, 2, 3, 4, and 5 cards, and a limited use card that is equivalent to a level 5 card that can only be used five times in Survival mode.",
-	"help.securitycraft.remote_access_mine.info": "The mine remote access tool will allow you to access mines remotely. Rightclick on a mine to 'bind' it to the tool. Rightclick in the air (with the tool equipped) to open the tool's GUI, which will allow you to activate, deactivate, or detonate any bound mines.",
+	"help.securitycraft.remote_access_mine.info": "The mine remote access tool will allow you to access mines remotely. Rightclick on a mine to 'bind' it to the tool. Rightclick in the air (with the tool equipped) to open the tool's GUI, which will allow you to activate, deactivate, detonate, or unbind any bound mines.",
 	"help.securitycraft.remote_access_sentry.info": "The sentry remote access tool will allow you to access sentries remotely. Rightclick on a sentry to 'bind' it to the tool. Rightclick in the air (with the tool equipped) to open the tool's GUI, which will allow you to change the mode of each bound sentry, or of all bound sentries at the same time.",
-	"help.securitycraft.bucket_f_water.info": "The fake water acts the same as water, expect it hurts you when touched.",
-	"help.securitycraft.bucket_f_lava.info": "The fake lava acts the same as lava, except it heals you instead of hurting you.",
+	"help.securitycraft.bucket_f_water.info": "The fake water acts the same as water, except it hurts you like lava when touched.",
```

Comment:
Harsh. Question: if something is placed in a block of fake water (coral, lily pad, sea pickle, etc), the water level of that block rises and mobs like fish no longer get hurt/die within that block. What actually happens there?

Reply:
I like the first one.
As for your question: I'm assuming that Minecraft waterlogs the block with normal water, as Fake Water is tagged as vanilla water to make slowing down entities possible.

Reply:
Yes, please take it off. It's not intended but also not fixable to my knowledge :P

Reply:
Yea probably not. I asked a dev just in case. If there’s a potential solution, will have Gef update you.

Reply:
Got a reply: in Java Edition, there’s something called a Liquids Layer, but it might be restricted to vanilla water only. Still your best shot if you ever get around to it.